app/views/assessments/viewGradesheet.html.erb

```diff
@@ -238,6 +246,61 @@ Interaction between gradesheet and annotations
 <span id="interaction_badge" class="badge" data-badge-caption="Feature Information"></span></a>
 </div>
 
+<script type="text/javascript">
+    var loadFunc = (elem) => {
```

Comment:
Is this function used anymore? Also, why are we using local storage here? There is also a console log that is unnecessary.